federation/pkg/kubefed/init/init.go

```diff
@@ -389,8 +389,6 @@ func createAPIServer(clientset *client.Clientset, namespace, name, image, creden
 		"--etcd-servers=http://localhost:2379",
 		"--service-cluster-ip-range=10.0.0.0/16",
 		"--secure-port=443",
-		"--token-auth-file=/etc/federation/apiserver/known_tokens.csv",
-		"--basic-auth-file=/etc/federation/apiserver/basic_auth.csv",
```

Comment:
oh sorry. We do not support these auth mechanisms in the new setup? That is fine for alpha. I assume you have plans to support it later.